### PR TITLE
Smart Contract Wiki

### PR DESCRIPTION
**Pull request by TRTLAlien**
Smart contract explained

### CODE REVIEW

File: guides/smart_contract.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# What is a Smart Contract?
+A **Smart Contract** is a peice of code that is stored in the blockchain network. It consists of conditions that everyone using the contract must agree on. Every computer in the network will get the same result and in turn people will know that the outcome is always correct.
```
Comment (ZedPea) [inline guides/smart_contract.md]: Typo: peice -> piece

File: guides/smart_contract.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
...f there is no incentive to seed people generally don’t, so what if their clients seed the most profitable data that needs the most iops for that pool? Like someone running a smart contract social network on our chain in private would need.Ok so if these encrypted blobs take a muktisig key to operate you can control access to it with iterating one time scripts where you send a small payment to initiate a script, you’re generated a session with your view key or whatever as part of the auth handshake with the script and then your client moves a few units with a certain payment ID attached to prove ownership of the view key maybe using the change address to watch the change come back, I haven’t fleshed that part out yetSo you use up your session of whatever you’re consuming in the script, whatever maybe betting on a basketball gameAnd the script lives on that addressNew address type of courseThe script conditionally parses data and hands out the result or computes the operationsWe do similar to ETH where we charge by operation with a finite resource, and then have computational units and storage units for the multiple use cases so there’s no taking advantage of the chain without paying
```
Comment (ZedPea) [inline guides/smart_contract.md]: Probably don't want this to be bold, it's a bit "In your face".

Also would be easier to read if split into paragraphs and the typos fixed.

**Comment by SoreGums** [comment]
how about like this - seems kind of important to mention it's a quote if it is a quote rather than jump straight into "I...."

https://gist.github.com/SoreGums/83565b862100fe6e310a6aa4cf11cd88

**Comment by SoreGums** [comment]
shall we just squash this 1 commit and do it?
since it is not in the sidebar or anywhere - ppl need to find it...

**Comment by Sajo811** [comment]
this should come under cryptocurrency fundamentals, maybe `crypto-funda` as the folder, so that it will come under that name in the sidebar

**Comment by SoreGums** [comment]
fail to type above - meant to say this article is mostly an idea dump, probably could go in meta, however, it is fine enough to go here and not be linked. so if needed someone can direct link to it if it comes up in conversation /biz/

**Comment by Sajo811** [comment]
agreed with what @SoreGums said, I would just like to say, move this out of the `guides` folder, make a new folder like `crypto-funda` and place it there

**Pull request closed without merging**